app/test_split_integer.py

```diff
@@ -1,21 +1,29 @@
+import pytest
+
 from app.split_integer import split_integer
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    value = 17
+    number_of_parts = 4
+    assert sum(split_integer(17, 4)) == value
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    parts = split_integer(32, 6)
+    assert parts.count(6) == 2 and parts.count(5) == 4
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    parts = split_integer(8, 1)
+    assert parts == [8]
```

Comment:
Code Style: Here too, avoid creating the `parts` variable. The assert statement could be written as: `assert split_integer(8, 1) == [8]`.